astropy/units/function/core.py

```diff
@@ -405,9 +412,31 @@ def to_string(self, format="generic"):
             # need to strip leading and trailing "$"
             self_str += rf"$\mathrm{{\left( {pu_str[1:-1]} \right)}}$"
         else:
-            self_str += f"({pu_str})"
+            pu_lines = pu_str.splitlines()
+            if len(pu_lines) == 1:
+                self_str += f"({pu_str})"
+            else:
+                # If the physical unit is formatted into a multiline
```

Comment:
Since you now support it, I think we might as well pass on `inline` (or rather `**kwargs`)?

Reply:
Yes, once https://github.com/astropy/astropy/pull/14393 is merged, I'll update.